services/gateway/util/src/main/java/org/eclipse/ditto/services/gateway/util/config/security/SubjectIssuerConfig.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2021 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.ditto.services.gateway.util.config.security;
+
+import java.util.List;
+
+/**
+ * Represents configuration for a {@link SubjectIssuer }, containing
```

Comment:
The `@link SubjectIssuer` won't work now because we are not in the same package and do not have that imported.
Suggesting:
```suggestion
 * Represents configuration for a {@link org.eclipse.ditto.model.policies.SubjectIssuer}, containing
```